task2.py

```diff
@@ -0,0 +1,37 @@
+"""Create unit tests for any program from previous homeworks."""
+import unittest
+
+
+def count_characters(s='kappa'):
+    """Function count and return the numbers of each character in a string."""
+    dct = {}
+    for symb in s:
+        dct[symb] = dct.get(symb, 0) + 1
+    return dct
+
+
+class TestCountCharacters(unittest.TestCase):
+    def test_normal(self):
+        res = count_characters('coromilly')
+        self.assertEqual(
+            res, {'c': 1, 'o': 2, 'r': 1, 'm': 1, 'i': 1, 'l': 2, 'y': 1})
+
+    def test_default_value(self):
```

Comment:
it has strange default value. i.e. we can call it with empty args and check assert raises